agent/js/package.json

```diff
@@ -2,9 +2,19 @@
   "name": "agent-protocol",
   "version": "0.1.0",
   "description": "API for interacting with Agent",
+  "homepage": "https://e2b.dev",
+  "license": "MIT",
+  "author": {
+    "name": "e2b",
+    "email": "hello@e2b.dev",
+    "url": "https://e2b.dev"
+  },
+  "bugs": "https://github.com/e2b-dev/agent-protocol/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/e2b-dev/agent-protocol"
```

Comment:
We want the url here to reflect the subdirectory with the package.

Reply:
Alright, that's changed to `https://github.com/e2b-dev/agent-protocol/tree/main/agent/js`.